packages/ui/src/components/va-switch/VaSwitch.vue

```diff
@@ -332,6 +332,11 @@ export default class VaSwitch extends mixins(
   #{&}__track-label {
     color: $white;
     margin: auto 0.5rem auto 2rem;
+    user-select: none;
+    -webkit-user-select: none;
```

Comment:
Pretty sure prefixes are auto-added (no need to provide them in css, `user-select: none`, should be enough).